Validation/HLTrigger/plugins/HLTGenValSource.cc

```diff
@@ -255,23 +272,23 @@ void HLTGenValSource::analyze(const edm::Event& iEvent, const edm::EventSetup& i
 // ------------ method called once each job just before starting event loop  ------------
 void HLTGenValSource::bookHistograms(DQMStore::IBooker& iBooker, const edm::Run& run, const edm::EventSetup& setup) {
   iBooker.setCurrentFolder(dirName_);
-
-  if (infoString_.back() == ',')
-    infoString_.pop_back();
-  infoString_ += "}";  // adding the closing bracked to the JSON string
   iBooker.bookString("HLTGenValInfo", infoString_);
+  if (infoString_ == "{}" || infoString_.empty()) {
+    std::cout << "infostr" << infoString_ << std::endl;
```

Comment:
std::cout is not thread safe, pleas use `edm::LogInfo` instead

Reply:
thanks not sure how that was in there, fixed